main/src/library/Fixpoint/Ast/Denotation.flix

```diff
@@ -16,6 +16,6 @@
 namespace Fixpoint/Ast {
     pub enum Denotation {
         case Relational,
-        case Latticenal
+        case Latticenal(Unit -> Boxed, Boxed -> Boxed -> Bool, Boxed -> Boxed -> Boxed, Boxed -> Boxed -> Boxed)
```

Comment:
Why does Latticenal take Boxed values opposed to v?

Reply:
You are right, it should be v's